lxd/cluster/connect.go

```diff
@@ -223,8 +223,12 @@ func SetupTrust(serverCert *shared.CertInfo, serverName string, targetAddress st
 	}
 
 	post := api.CertificatesPost{
-		CertificatePut: cert.CertificatePut,
```

Comment:
Shouldn't this be using `post.SetWritable(cert.CertificatePut)`?

Reply:
This is Post structure which does not have SetWritable() currently. 
And Post is expecting base64 encoded certificate, while Put is not.
Wouldn't it get confusing if we add SetWritable() function?

Reply:
ah ok thanks